src/scrape_up/github/respository.py

```diff
@@ -403,4 +403,20 @@ def get_environment(self):
             return {
                 "data": None,
                 "message": message,
+            }
+
+    def get_branch(self):
+        data = requests.get(f"https://github.com/{self.username}/{self.repository}/branches")
+        data = BeautifulSoup(data.text, "html.parser")
+        try:
+            branch = data.find_all(class_="branch-name css-truncate-target v-align-baseline width-fit mr-2 Details-content--shown")
+            allBranches = []
+            for branchNames in branch:
+                allBranches.append(branchNames.text.strip())
+            return allBranches
```

Comment:
Also, change the return type in the `try` block.

Reply:
> Also, change the return type in the `try` block.

Okk, sure.

Reply:
Done with the bug fixes. :)